ksqldb-functional-tests/src/test/resources/query-validation-tests/elements.json

```diff
@@ -73,6 +94,27 @@
       "inputs": [{"topic": "input", "value": {"c1": 4}}],
       "outputs": [{"topic": "OUTPUT", "value": {"C1": 4}}]
     },
+    {
+      "name": "validate schema id without elements OK - PROTOBUF",
+      "statements": [
+        "CREATE STREAM INPUT WITH (kafka_topic='input', value_format='PROTOBUF');",
```

Comment:
I don't see schema id in this statement?

Reply:
Good catch!